docs/man/docker.1.md

```diff
@@ -34,6 +34,9 @@ unix://[/path/to/socket] to use.
    The socket(s) to bind to in daemon mode specified using one or more
    tcp://host:port, unix:///path/to/socket, fd://* or fd://socketfd.
 
+**--add-registry**=[]
+  Each given registry will be prepended to a list of registries queried during image pulls or searches. The last registry given will be queried first. They will be treated as insecure. Registry mirrors won't apply to them.
```

Comment:
I think that the `--registry-mirror` option queried in the order given - if that's the case, whats the reason this option starts at the last registry given?

Should we really default to insecure registry?